.ci/magic-modules/generate-terraform.sh

```diff
@@ -30,7 +30,7 @@ bundle exec compiler -p products/resourcemanager -e terraform -o "${GOPATH}/src/
 bundle exec compiler -p products/redis -e terraform -o "${GOPATH}/src/github.com/terraform-providers/terraform-provider-google/"
 
 # Resources that were already using beta APIs before they started being autogenerated
-bundle exec compiler -v beta -p products/compute -t Address,Subnetwork,ForwardingRule -e terraform -o "${GOPATH}/src/github.com/terraform-providers/terraform-provider-google/"
+bundle exec compiler -v beta -p products/compute -t Address,Subnetwork,ForwardingRule,RegionDisk -e terraform -o "${GOPATH}/src/github.com/terraform-providers/terraform-provider-google/"
```

Comment:
This is getting a bit untenable - any ideas?

Reply:
In the future, we won't have this list at all- it'll put all resources at their beta versions into terraform-provider-google-beta, so I'm pretty ok with leaving it as-is as a temporary thing. I can pretty easily factor out the list into a variable though if you'd like.

Reply:
I could imagine some solutions to keep you from having to type it at all (we never want to not generate something at beta if it's got beta support in `api.yaml`), but if it's not bugging you let's live with it for now.